vignettes/why-it-works.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: "Interval censoring and truncation in epidemiological delay modelling"
+format: html
+bibliography: library.bib
+---
+
+# Introduction {#introduction}
+
+Time to event analysis, also known as survival analysis, concerns estimating the distribution of delay times between events. A distinctive feature of the field are the methodological techniques used to deal with the missing data problems common in data sets of delay times [@leung1997censoring]. In `primarycensoreddist` we focus on two particular missing data problems:
+
+-   **Interval censoring**. The primary (start) time and/or the secondary (end) time of the delay is unobserved but known to be within an interval.
+-   **Truncation**. Delay data items are only contained in the data set conditional on aspect of the data point, for example if the secondary event is the observation time which add the data item to the data set.
```

Comment:
```suggestion
-   **Truncation**. Delay data items are only contained in the data set conditional on an aspect of the data point, for example, if the secondary event is the observation time which adds the data item to the data set.
```

I'm not totally clear what you were going for here and it doesn't totally make sense to me even with edits. I've left it for you vs writing a def from my view point as I think its helpful if it comes from your perspective vs repeating mine from i.e. the getting started and other papers etc.